packages/vitest/src/node/browser/webdriver.ts

```diff
@@ -59,6 +63,7 @@ export class WebdriverBrowserProvider implements BrowserProvider {
       capabilities: {
         'browserName': this.browser,
         'wdio:devtoolsOptions': { headless: options.headless },
+        ...this.providerOptions,
```

Comment:
We should override browserName and `wdio:devtoolsOptions.headless`

Reply:
Do you mean should or shouldn't? Right now it will overwrite all former if present in `this.providerOptions`